Workbooks/VisualizationDemo.json

```diff
@@ -0,0 +1,523 @@
+                  {
+                    "columnMatch": "RecommendationSeverity",
+                    "formatter": 18,
+                    "formatOptions": {
+                      "thresholdsOptions": "colors",
+                      "thresholdsGrid": [
+                        {
+                          "operator": "==",
+                          "thresholdValue": "High",
+                          "representation": "redBright",
+                          "text": "{0}{1}"
+                        },
+                        {
+                          "operator": "==",
+                          "thresholdValue": "Medium",
+                          "representation": "yellow",
+                          "text": "{0}{1}"
+                        },
+                        {
+                          "operator": "==",
+                          "thresholdValue": "Low",
+                          "representation": "blue",
+                          "text": "{0}{1}"
+                        },
+                        {
+                          "operator": "==",
+                          "thresholdValue": "Informational",
+                          "representation": "gray",
+                          "text": "{0}{1}"
+                        },
+                        {
+                          "operator": "Default",
+                          "thresholdValue": null,
+                          "representation": "blue",
+                          "text": "{0}{1}"
+                        }
+                      ]
+                    }
+                  },
+                  {
+                    "columnMatch": "Link",
+                    "formatter": 7,
+                    "formatOptions": {
+                      "linkTarget": "GenericDetails",
+                      "linkIsContextBlade": true
+                    }
+                  }
+                ],
+                "labelSettings": [
+                  {
+                    "columnId": "AssessedResourceId",
+                    "label": "Resource"
+                  },
+                  {
+                    "columnId": "RecommendationName",
+                    "label": "Recommendation"
+                  },
+                  {
+                    "columnId": "RecommendationState",
+                    "label": "Status"
+                  },
+                  {
+                    "columnId": "RecommendationSeverity",
+                    "label": "Severity"
+                  },
+                  {
+                    "columnId": "Link"
+                  }
+                ]
+              }
+            },
+            "name": "query - 5"
+          }
+        ]
+      },
+      "conditionalVisibility": {
+        "parameterName": "Tab",
+        "comparison": "isEqualTo",
+        "value": "Personalization"
+      },
+      "name": "Personalization"
+    }
+  ],
+  "styleSettings": {},
+  "fromTemplateId": "VisualizationDemo",
```

Comment:
This should be - sentinel-VisualizationDemo

Reply:
Edited the file to reflect the comment